src/components/Camera/index.js

```diff
@@ -77,111 +61,62 @@ class CameraPure extends React.Component<CameraPureType> {
   }
 
   render() {
-    const {method, autoCapture, title, subTitle, onUploadFallback, onFallbackClick,
-      onUserMedia, faceCaptureClick, webcamRef, isFullScreen, onWebcamError, i18n} = this.props;
+    const {method, title, subTitle, onUploadFallback, onFallbackClick,
+      onUserMedia, webcamRef, isFullScreen, onWebcamError, i18n, liveness: video} = this.props;
 
     return (
       <div className={style.camera}>
         <Title {...{title, subTitle, isFullScreen}} smaller={true}/>
         <div className={classNames(style["video-overlay"], {[style.overlayFullScreen]: isFullScreen})}>
           <Webcam
             className={style.video}
-            audio={false}
+            audio={!!video}
             height={720}
             {...{onUserMedia, ref: webcamRef, onFailure: onWebcamError}}
           />
           <Overlay {...{method, isFullScreen}}/>
-          <UploadFallback {...{onUploadFallback, onFallbackClick, method, i18n}}/>
+          { !video && <UploadFallback {...{onUploadFallback, onFallbackClick, method, i18n}}/> }
         </div>
-        { autoCapture ? '' : <CaptureActions handeClick={faceCaptureClick} {...{i18n, isFullScreen}}/>}
       </div>
     )
   }
 }
 
-type CameraType = {
-  ...CameraCommonType,
-  onScreenshot: Function,
-  trackScreen: Function,
-}
-
 type CameraStateType = {
   hasWebcamAccess: boolean,
   hasSeenPermissionsPrimer: boolean,
 }
 
 export default class Camera extends React.Component<CameraType, CameraStateType> {
-  webcam: ?React$ElementRef<typeof Webcam> = null
-  interval: ?Visibility
-
-  state:CameraStateType = {
+  state: CameraStateType = {
     hasWebcamAccess: false,
     hasSeenPermissionsPrimer: false,
   }
 
-  capture = {
-    start: () => {
-      this.capture.stop()
-      this.interval = Visibility.every(1000, this.screenshot)
-    },
-    stop: () => Visibility.stop(this.interval),
-    once: () => {
-      this.screenshot()
-    }
-  }
-
   constructor (props: CameraType) {
```

Comment:
minor nitpick: If you're not doing anything except calling super(props), this constructor can be dropped :)